waku/v2/node/wakunode2.nim

```diff
@@ -138,28 +181,38 @@ proc new*(T: type WakuNode, nodeKey: crypto.PrivateKey,
   let
     rng = crypto.newRng()
     hostAddress = tcpEndPoint(bindIp, bindPort)
+    wsHostAddress = tcpEndPoint(bindIp, wsbindPort) & addWsFlag
     announcedAddresses = if extIp.isNone() or extPort.isNone(): @[]
-                         else: @[tcpEndPoint(extIp.get(), extPort.get())]
+                         else: @[tcpEndPoint(extIp.get(), extPort.get()), 
+                         tcpEndPoint(extIp.get(), extPort.get()) & addWsFlag]
     peerInfo = PeerInfo.init(nodekey)
     enrIp = if extIp.isSome(): extIp
             else: some(bindIp)
     enrTcpPort = if extPort.isSome(): extPort
                  else: some(bindPort)
     enr = createEnr(nodeKey, enrIp, enrTcpPort, none(Port))
   
-  info "Initializing networking", hostAddress,
+  info "Initializing networking", hostAddress, wsHostAddress,
                                   announcedAddresses
   # XXX: Add this when we create node or start it?
-  peerInfo.addrs.add(hostAddress) # Index 0
+  # Index 0
+  if transport == "ws":
+    peerInfo.addrs.add(wsHostAddress)
+  else:
+    peerInfo.addrs.add(hostAddress)
+  
   for multiaddr in announcedAddresses:
     peerInfo.addrs.add(multiaddr) # Announced addresses in index > 0
   
-  var switch = newStandardSwitch(
-    some(nodekey),
-    hostAddress,
-    transportFlags = {ServerFlags.ReuseAddr},
-    rng = rng,
-    maxConnections = maxConnections)
+  var switch = newWakuSwitch(some(nodekey), wsHostAddress, 
```

Comment:
I think the existing formatting is fine and more clear, this change seems unrelated to PR